submissions/hadizaSaadu/HTML/E-commerce.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>product-page</title>
+  </head>
+  <body>
+    <h1>DEEZAH'S CLOSET</h1>
+    <h3>
+      <i>Home of modest wears . . .</i>
+    </h3>
+
+    <a href="#">HOME</a>
+
+    <a href="#">ABOUT US</a>
+    <a href="#">CONTACT</a>
+    <a href="#">FAQ</a>
+
+    <!--THIS SECTION IS FOR PRODUCTS-->
+    <section>
+      <h3>PRODUCTS SECTION</h3>
+      <table>
+        <tr>
+          <td>
+            <img
+              src="images/products/exclusiveabaya.jpg"
+              width="300"
+              height="450"
+            />
+
+            <p><strong>Comfy Exclusive Abaya</strong></p>
+            <p><strong>Price: #15,000</strong></p>
+            <p>
+              <strong>Product Description</strong><br />
+              this piece has been carefully designed to suit the need of all
+              modesty lovers,<br />it is a free size and fits for all
+              occassions, <br />
+              you would be missing out a great deal if you do not have this in
+              your closet!!!.
+            </p>
+          </td>
+          <td>
+            <img
+              src="images/products/umbrellastyle.jpg"
+              width="300"
+              height="450"
+            />
+
+            <p><strong>Comfy Umbrella Abaya</strong></p>
+            <p><strong>Price: #10,000</strong></p>
+            <p>
+              <strong>Product Description</strong><br />
+              this piece has been carefully designed to suit the need of all
+              modesty lovers,<br />it is a free size and fits for all
+              occassions, <br />
+              you would be missing out a great deal if you do not have this in
+              your closet!!!.
+            </p>
+          </td>
+        </tr>
+      </table>
+    </section>
+
+    <h3>CHECK OUT OTHER AVAILABLE PRODUCTS BELOW:</h3>
+    <ol>
+      <li><a href="#">Casual Abaya</a></li>
+      <li><a href="#">Egyptian Model</a></li>
+      <li><a href="#">Nursing Abaya</a></li>
+    </ol>
+
+    <footer>
+      <h3>CONTACTS US</h3>
+
+      <a href="https://www.facebook.com">
+        <img src="images/logos/Facebook.png" width="30" height="40" />
```

Comment:
@bint-saad @bint-saad Remove the width and height

Reply:
done!!